### PR TITLE
adding forking guide

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**What**
Adding a guide for forking to the existing git tutorial

ping  @dominikkawka @damienomurchu @maciejmarchel12 @AdamLalor for review.

### CODE REVIEW

**Review by dominikkawka — COMMENTED** [review]
maybe change "How to fork?" to "How do I fork?" or maybe drop the q.mark

**Comment by laurafitzgerald** [comment]
@dominikkawka good point, have updated this.

File: topic04-git-introduction/book-git-tutorial/05.Forking.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+## What is forking?
+
+To fork a repository means to get your own version/copy of the repository, and to be able to edit it without affecting the original repo (repository). 
+
+## How to fork?
+
+. You first go onto someone’s github profile
+. Click on one of their repositories 
+. Then click the fork button,
+. Which is usually located below your profile picture dropdown
```
Comment (AdamLalor) [inline topic04-git-introduction/book-git-tutorial/05.Forking.md]: Why is this on a separate line and have a capital "W"

**Review by dominikkawka — APPROVED** [review]

File: topic04-git-introduction/book-git-tutorial/05.Forking.md (hunk 1 above)
Comment (laurafitzgerald) [reply topic04-git-introduction/book-git-tutorial/05.Forking.md]: @AdamLalor well spotted. I took the content from the doc without review. Will update it thanks

Comment (laurafitzgerald) [reply topic04-git-introduction/book-git-tutorial/05.Forking.md]: @AdamLalor updated if you have time to review again please

**Review by AdamLalor — APPROVED** [review]
Looks Good

**Comment by laurafitzgerald** [comment]
@AdamLalor would you mind sticking on an Approved status on this pr

**Review by AdamLalor — APPROVED** [review]

**Review by maciejmarchel12 — APPROVED** [review]

File: topic04-git-introduction/book-git-tutorial/05.Forking.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## How to fork
+
+. You first go onto someone’s github profile
+. Click on one of their repositories 
+. Then click the fork button which is usually located below your profile picture dropdown (see image)
+
+ ![](https://github.com/feedhenry/student-help-guide-content/tree/master/topic04-git-introduction/book-git-tutorial/img/fork.png) 
+
+ Remember (You can’t fork your own repository!)
```
Comment (JameelB) [inline topic04-git-introduction/book-git-tutorial/05.Forking.md]: why is `(You can't fork your own repository!)` in brackets?

**Review by damienomurchu — COMMENTED** [review]
I like the section, and think the content is great. 

Do you think we should have a reference on how to clone the forked repo, and how to add the original project as a remote to your repo

**Comment by laurafitzgerald** [comment]
@damienomurchu  yes agreed this is a good idea. I've added how to add a remote. We already covered cloning in a previous lab so left that out. Could you review this when you get a chance :)

**Review by maciejmarchel12 — APPROVED** [review]

**Comment by laurafitzgerald** [comment]
@JameelB could you do a quick review of this so that I can get it merged.

File: topic04-git-introduction/book-git-tutorial/05.Forking.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Forking on GitHub
+
+## What is forking?
+
+To fork a repository means to get your own version/copy of the repository, and to be able to edit it without affecting the original repo (repository). 
+
+## How to fork
+
+. You first go onto someone’s github profile
```
Comment (JameelB) [inline topic04-git-introduction/book-git-tutorial/05.Forking.md]: I think these are supposed to be a list but they're not rendered correctly

File: topic04-git-introduction/book-git-tutorial/05.Forking.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+. Click on one of their repositories 
+. Then click the fork button which is usually located below your profile picture dropdown (see image)
+
+ ![](https://github.com/feedhenry/student-help-guide-content/tree/master/topic04-git-introduction/book-git-tutorial/img/fork.png)
```
Comment (JameelB) [inline topic04-git-introduction/book-git-tutorial/05.Forking.md]: Not sure if this image will render using http link. You might need to use relative paths

**Review by JameelB — CHANGES_REQUESTED** [review]
Looks good, just additional small changes

**Comment by laurafitzgerald** [comment]
@JameelB thanks for the review. Made updates as suggested. could you take a look again when you get a chance.

**Review by JameelB — DISMISSED** [review]

**Comment by laurafitzgerald** [comment]
@JameelB merged thanks. do we need to take some step to update the site now?

**Comment by JameelB** [comment]
@laurafitzgerald Yes, site will need to be generated. I can generate that for you. Will ping you on PR

**Comment by laurafitzgerald** [comment]
@JameelB thanks :)

**Pull request merged**